app/vmalert/remotewrite/remotewrite.go

```diff
@@ -165,6 +176,16 @@ func (c *Client) run(ctx context.Context) {
 			case <-ctx.Done():
 				shutdown()
 				return
+			case <-c.debugflushCh:
```

Comment:
This looks too brittle. Can we try to make it simpler by introducing a new RW implementation that would satisfy the same interface but would just have a blocking Push method? This would be simpler and more robust than extending the existing RW client

Reply:
Sounds reliable, add a debug rw client now.